### PR TITLE
Add Nightly and Release workflows for FirebasePerformance.

### PR DESCRIPTION
**Pull request by visumickey**
Fixes https://github.com/firebase/firebase-ios-sdk/issues/7266

### CODE REVIEW

**Comment by paulb777** [comment]
I'll work on a fix for #7354 on Monday and unblock the CI here

**Comment by paulb777** [comment]
Restarted after updating SpecsStaging with 7.5.0 based on #7371

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -470,3 +470,42 @@ jobs:
       with:
         name: quickstart_artifacts_storage
         path: quickstart-ios/
+
+  performance_quickstart:
+    # Don't run on private repo unless it is a PR.
+    if: (github.repository == 'Firebase/firebase-ios-sdk' && github.event_name == 'schedule') || github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch'
+    needs: buildup_SpecsReleasing_repo
```
Comment (paulb777) [inline .github/workflows/release.yml:477]: buildup_SpecsTesting_repo

Comment (visumickey) [reply .github/workflows/release.yml]: Done!

**Review by paulb777 — APPROVED** [review]
LGTM on green CI

**Review by jeremyjiang-dev — APPROVED** [review]

**Pull request merged**